src/components/Widgets/Form/AddressSelect.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import shortid from 'shortid';
+import Identicon from '../../Identicon'
+import { combineWallets } from '../../../util/helpers';
+
+export default class AddressSelect extends Component {
+  static displayName = 'AddressSelect';
+
+  static propTypes = {
+    disabled: PropTypes.bool,
+    wallets: PropTypes.object,
+    walletContracts: PropTypes.object,
+    identiconSize: PropTypes.string,
```

Comment:
I think we can remove the option to set `Identicon` size, and hard code it as `tiny`. I don't believe this component is used elsewhere or needs this type of configurability. Correct me if I'm wrong.

Reply:
agreed, but i do like your initiative kiel of adding configurable options :) can be useful, but agree with marc let's do it when we find the need